src/main/java/com/google/neighborgood/servlets/TaskServlet.java

```diff
@@ -65,18 +66,27 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
       response.sendError(HttpServletResponse.SC_BAD_REQUEST, "Location coordinates are missing");
     }
 
+    if (request.getParameterMap().containsKey("miles")) {
+      double miles = 5;
```

Comment:
Can we define a static final variable `DEFAULT_MILE_RADIUS` and set it to 5? This will make the code more understandable

Reply:
Good idea, I'll make these changes, thanks!